script/src/syscalls/load_cell.rs

```diff
@@ -67,6 +75,7 @@ impl<'a, Mac: SupportMachine> Syscalls<Mac> for LoadCell<'a> {
             return Ok(true);
         }
         let cell = cell.unwrap();
+        let output = self.store.lazy_load_cell_output(&cell);
```

Comment:
This is just a minor thing so feel free to ignore it and leave it to a future PR: I noticed right now we are still using `flat_serializer` for serializing CellOutput in the store. And right now, the life of a CellOutput is:

1. When inserting the block, a CellOutput is serialized using `flat_serializer` and put into store
2. Here when we are loading the cell, we first load it and deserialize it using `flat_serializer`
3. Then the CellOutput is serialized again using flatbuffers, and VM reads the section it needs.

What if we just use flatbuffer to serialize CellOutput? Pairing it with partial read in rocksdb, we can only read the exact bytes required by CKB VM in the syscall from rocksdb, and return those directly to the VM, no serialization/deserialization steps are needed here in this way.

I cannot promise the percentage of speedup we can get by implementing this, but what I can promise is: if we can make the change here, we won't need the comments below saying load cell is an expensive operation which should be avoided, and we can revise the cycle costs of this syscall, enabling totally new paradigms in our scripts which is both easier to write and cost effective.

Thoughts?

Reply:
Totally agree, except CFB is not ready yet. In fact, we're using `bincode` for serialize and I have no idea how to achieve this with `bincode`.

Reply:
Sounds like a good future PR candidate.